pages/wikilist/components/UserList/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import styles from './styles.module.scss';
```

Comment:
1. 멘토링 때 멘토님이 말씀해주신 것처럼 React는 import를 안 해줘도 될 것 같아요!
2. styles를 import 할 때는 절대경로를 사용해서 import 해주세요!
```suggestion
import { useState, useEffect, useCallback } from 'react';
import styles from '@/pages/wikilist/components/UserList/styles.module.scss';
```
3. 컴포넌트 폴더명도 `camelCase`를 사용하고 있습니다.
UserList -> userList, UserCard -> userCard ...